src/utils/tasks.js

```diff
@@ -132,7 +132,9 @@ export function formatDuration (value, allowZeros = false, timingOption = true)
 
 function formatRSS (value) {
   // Format the max RSS value in a human-readable format
-  if (value / 1024 < 1000) {
+  if (value === 0 || value === undefined || value === null) {
+    return undefined
```

Comment:
https://github.com/cylc/cylc-uiserver/pull/675#issuecomment-3480728965

> I don't think that fix is valid - the UI should treat 0 as 0 seconds, and null/undefined should be treated as empty. So the uiserver should send null when there is no data rather than 0. Some tasks may run in ~0 seconds so 0 is a valid normal value.